app/models/cloud_volume.rb

```diff
@@ -9,15 +9,13 @@ class CloudVolume < ApplicationRecord
   belongs_to :availability_zone
   belongs_to :cloud_tenant
   belongs_to :base_snapshot, :class_name => 'CloudVolumeSnapshot', :foreign_key => :cloud_volume_snapshot_id
-  has_many   :cloud_volume_snapshots
-  has_many   :attachments, :class_name => 'Disk', :as => :backing
+  has_many :cloud_volume_snapshots
+  has_many :attachments, :class_name => 'Disk', :as => :backing
+  has_many :hardwares, :through => :attachments
```

Comment:
This is a weird relationship...I get why it's being done, but I just don't see how it would be used outside of the vms reference.  I'm wondering if there's a way to mark it "private"